### PR TITLE
Add Windows support

### PR DESCRIPTION
**Pull request by tritao**
This pull request adds support for running `react-native-tts` with `react-native-windows`.

For context, I've also opened a PR which updates the example to the latest `react-native` 0.63 and adds an Windows example: https://github.com/themostaza/react-native-tts-example/pull/2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,55 @@ react-native link react-native-tts
...
 
+#### Windows
+
+1. In `windows/myapp.sln` add the `RNTTS` project to your solution:
+
+   - Open the solution in Visual Studio 2019
+   - Right-click Solution icon in Solution Explorer > Add > Existing Project
+   - Select `node_modules\react-native-tts\windows\RNTTS\RNTTS.vcxproj`
+
+2. In `windows/myapp/myapp.vcxproj` ad a reference to `RNTTS` to your main application project. From Visual Studio 2019:
```
Comment (stmoy) [inline README.md:35]:
```suggestion
2. In `windows/myapp/myapp.vcxproj` add a reference to `RNTTS` to your main application project. From Visual Studio 2019:
```

File: README.md [hunk 2]
```diff
@@ -24,6 +24,55 @@ react-native link react-native-tts
...
+
+4. In `app.cpp` add `PackageProviders().Append(winrt::RNTTS::ReactPackageProvider());` before `InitializeComponent();`.
+
+### Windows MediaElement
+
+On Windows, `react-native-tts` needs a media element in the visual tree to output the speech stream.
+It will look for a XAML `MediaElement` named `RNTextToSpeech` and use it.
+This needs to be added to the XAML tree of the screens where `react-native-tts` is used.
+
+As an example, in `windows/myapp/MainPage.xaml` from the `react-native-tts` app template this can be done by adding a XAML `Grid` with an invisible `MediaElement` alongside the `ReactRootView`. Change `windows/myapp/MainPage.xaml` from:
```
Comment (stmoy) [inline README.md:50]:
```suggestion
As an example, in the `windows/myapp/MainPage.xaml` file from the `react-native-tts` app template, you can add a XAML `Grid` with an invisible `MediaElement` alongside the `ReactRootView`. Change `windows/myapp/MainPage.xaml` from:
```

File: README.md [hunk 3]
```diff
@@ -24,6 +24,55 @@ react-native link react-native-tts
...
+  />
+</Page>
+```
+to
+```xaml
+<Page
+  ...
+  >
+  <Grid>
+    <MediaElement x:Name="RNTextToSpeech" />
```
Comment (stmoy) [inline README.md:67]: MediaElement has been replaced by MediaPlayerElement - [link to MSDN](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.mediaelement?view=winrt-19041#remarks). We discourage use of MediaElement; can you double check that this still works assuming MediaPlayerElement is used instead?

**Comment by tritao** [comment]
Hi @stmoy, thanks for the review feedback, I've further refined the code on the PR, it's now using a `MediaPlayer` directly instead of going through either a `MediaElement` or `MediaPlayerElement`. This simplifies setup and usage of the module, it no longer requires the addition of the element to the XAML tree.

**Comment by ak1394** [comment]
Thansk for the contribution @tritao ! It looks good, and I intend to merge and release a new version with it soon. Would you be able to test master on windows before the release? I can't test windows buld myself.

**Comment by tritao** [comment]
> Thansk for the contribution @tritao ! It looks good, and I intend to merge and release a new version with it soon. Would you be able to test master on windows before the release? I can't test windows buld myself.

Sure, let me know when its ready and I'll test it.

**Comment by tritao** [comment]
@ak1394 By the way, I'm also working on CI and automated testing for this module, would you (and @themostaza) be open to merging the example in https://github.com/themostaza/react-native-tts-example in this repo for easier setup?

**Comment by ak1394** [comment]
Yes, it would be great to have some CI and tests

**Comment by ak1394** [comment]
Hi  @tritao I've merged PR, could you test it and let me know if it looks good?

**Comment by tritao** [comment]
@ak1394 Thanks for the merge, will do.

**Pull request merged**